tests/integration/tests/test_create_campaign.py

```diff
@@ -0,0 +1,112 @@
+"""
+This is an example automated test to newsletter generator application
+Our automated test will do the following:
+    #Open Qxf2 newsletter generator application
+    #Fill the details of create newsletter section.
+"""
+import os,sys,time
+from typing_extensions import runtime
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+from page_objects.PageFactory import PageFactory
+from utils.Option_Parser import Option_Parser
+import conf.create_newsletter_conf as conf
+import pytest
+
+@pytest.mark.GUI
+def test_create_campaign(test_obj):
+
+    "Run the test"
+    #try:
+    if True:
+        #Initalize flags for tests summary
+        expected_pass = 0
+        actual_pass = -1
+        #Create a test object for edit an article
+        test_obj = PageFactory.get_page_object("create newsletter page")
+        #Set start_time with current time
+        start_time = int(time.time())
+        
+        #Get the test details from the conf file
+        email = conf.email
+        password = conf.password
+        title = conf.title
+        category = conf.category
+        subject = conf.subject
+        opener = conf.opener
+        preview = conf.preview
+        
+        #Create Newsletter
+        result_create_newsletter = test_obj.create_newsletter(category,title)
```

Comment:
Rename this to select articles

But still abstract all these into one `create_newsletter` method that selects articles, sets opener, etc.